common/men_at_arms_types/00_maa_types.txt

```diff
@@ -423,6 +423,33 @@ massive_aerial_unit = {
 	
```

Comment:
You didn't rename the unit type.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/119843858-fbd39580-bf18-11eb-9189-b16f23d45486.png)

</details>